migrating/cluster-wide/ossm-migrating-cluster-wide.adoc

```diff
@@ -92,14 +93,18 @@ You can proceed to complete the migration and remove {SMProduct} 2 resources.
 Before creating a default revision tag and relabelling the namespaces, you must migrate all remaining workload namespaces, including gateways.
 ====
 
-// The following tasks are for migration using the Istio injection label with cert-manager
+// This sets the context to build the tasks for cluster-wide migration using the Istio injection label with cert-manager
+:context: cw-injection-cm
 
-include::modules/ossm-migrating-a-cluster-wide-deployment-using-the-istio-injection-label-with-cert-manager.adoc[leveloffset=+1]
+include::modules/ossm-migrating-a-cluster-wide-deployment-using-the-istio-injection-label.adoc[leveloffset=+1]
 
-include::modules/ossm-migrating-workloads-using-the-istio-injection-label-with-cert-manager.adoc[leveloffset=+2]
+// include::modules/ossm-migrating-a-cluster-wide-deployment-using-the-istio-injection-label-with-cert-manager.adoc[leveloffset=+1]
```

Comment:
Could you now delete this content? Commenting it out here means that it's not being used here, right?

Reply:
Correct. Thanks for catching that @nrfox! I deleted two instances and pushed the changes to the PR. I'll squash it later because some people like to see the various submissions that were made during the review.

Reply:
done